main/menus/mainMenu.js

```diff
@@ -39,9 +39,9 @@ const createNewProj = () => {
   }
 }
 
+
 const openExistingProject = (windowObj) => {
 
-  exec(`killall node`);
   global.mainWindow.webContents.send('closeSim', 'helloworld');
   global.newProj = false;
   //opens a directory
```

Comment:
do you need 'this' in this.windowObj? if this is not referring to the func parameter you should probably change the parameter name

Reply:
True we do not technically need a parameter however we would need the "this" keyword on the windowObj otherwise it would not be defined. The reason is because I put the logic into an external function and windowObj is defined as a parameter inside of menu.

Reply:
hmm not too sure what you mean by this, are you in codesmith right now?

Reply:
Yep

Reply:
im downstairs by the couch if you wanna show me the code

Reply:
Oh I am in Codesmith NY

Reply:
gotcha, well from what i understand it sounds like you should prolly pass the windowObj as an argument to openExistingProject to keep this function independent of any other function, or define this function as a helper function inside the function you are using it.